### PR TITLE
Project Toolbar Redesign

### LINKED ISSUE #81: The dashboard and tool bar should be clean and responsive single page views

### PR DESCRIPTION
**Pull request by milos381**
### Summary

Toolbar styling finished

### Type of Change

Please delete options that are not relevant.

-   [ ] Bug fix (non-breaking change which fixes an issue)

### Issues

closes #81

### CODE REVIEW

**Review by shalarewicz — COMMENTED** [review]
The new toolbar looks great. Will approve after comments in #82 are addressed and merged as this contains those changes as well.

File: public/styles.css [hunk 1]
```diff
@@ -383,51 +384,96 @@ section#pageThree {
...
     top: 0px;
-    z-index: 10;
-    background-color: #f5f5f5;
-    padding-top: 75px;
+    left: 0;
+    color: #324da6;
+    background-color: #fff;
+    margin-top: 64px;
+    border-bottom: 1px solid #324da6;
+    transition: all 0.2s ease-out;
```
Comment (feiw101) [inline public/styles.css:399]: Just a quick suggestion, feeling like we can slow it down a little for a smoothier movement; Whether slow it down or not, it's already perfect...

**Review by feiw101 — CHANGES_REQUESTED** [review]

**Review by donjewey — APPROVED** [review]
Approved to allow swifter merging before launch, once pagination is updated and merged someone else can approve and merge this branch.

File: client/components/ProjectItem.tsx [hunk 2]
```diff
@@ -25,15 +25,15 @@ export default function ProjectItem({
             </button>
             {dropdown && (
                 <div className="projectContent">
-                    <div>
+                    <div className="apiDiv">
                         <b>API Key:</b> {project.apiKey}
```
Comment (evanmcneely) [inline client/components/ProjectItem.tsx:29]: Can you display `project_id` here? When discussing with Flora it sounded like you had done this but it's not here (unless it's in a different PR). Either way it needs to be visible or no one can setup the logger to use the app.

**Review by evanmcneely — CHANGES_REQUESTED** [review]
Just one comment from me but it's important

**Review by feiw101 — APPROVED** [review]

File: public/styles.css (hunk 1 above)
Comment (milos381) [reply public/styles.css:399]: Ok :)

File: client/components/ProjectItem.tsx (hunk 2 above)
Comment (milos381) [reply client/components/ProjectItem.tsx:29]: Yup I will do it in mp/main-responsive

**Review by evanmcneely — APPROVED** [review]

File: client/components/ProjectItem.tsx (hunk 2 above)
Comment (evanmcneely) [reply client/components/ProjectItem.tsx:29]: Yep it's all good there.

**Pull request merged**